src/transaction/log_recovery.c

```diff
@@ -1172,23 +1162,34 @@ log_recovery_redo (THREAD_ENTRY * thread_p, log_recovery_context & context)
 	      }
 	  }
 
+	  /* Read the log record information from log and then if redo is required apply it */
+
+	  const LOG_LSA rcv_lsa = redo_context.m_reader.get_lsa ();	/* Address of redo log record */
```

Comment:
Moved here from each case; please check for mistakes.

Reply:
`get_lsa` is not used for `2PC` branches, neither for ABORT/COMMIT.